cmd/krew/cmd/list.go

```diff
@@ -56,20 +134,55 @@ Remarks:
 				return nil
 			}
 
-			// print table
-			var rows [][]string
-			for p, version := range plugins {
-				rows = append(rows, []string{p, version})
+			if *output.OutputFormat == "wide" {
+				// print table
+				var rows [][]string
+				for p, version := range plugins {
+					rows = append(rows, []string{p, version})
+				}
+				rows = sortByFirstColumn(rows)
+				return printTable(os.Stdout, []string{"PLUGIN", "VERSION"}, rows)
 			}
-			rows = sortByFirstColumn(rows)
-			return printTable(os.Stdout, []string{"PLUGIN", "VERSION"}, rows)
+
+			if inArray(*output.OutputFormat, output.AllowedFormats()) {
+				objs := []Entry{}
+				for plugin, version := range plugins {
+					obj := Entry{plugin, version}
+					objs = append(objs, obj)
+				}
+				objs = sortByName(objs)
+				for _, obj := range objs {
+					p, err := output.ToPrinter()
+					if err != nil {
+						return err
+					}
+					err = p.PrintObj(obj, os.Stdout)
+					if err != nil {
+						return err
+					}
+				}
+				return nil
+			}
+			return errors.New("unsupported output format specified")
 		},
 		PreRunE: checkIndex,
 	}
 
+	listCmd.Flags().StringVarP(&outputFormat, "output", "o", outputFormat, "_")
```

Comment:
I just needed a placeholder as I didn't have a description for the flag.

> also why is the default value listed as outputFormat which is an empty string at this point?

Seemingly no reason, it can and probably should be changed to "". Changed.